tools/topology/CMakeLists.txt

```diff
@@ -1,18 +1,19 @@
 set(SOF_TOPOLOGY_BINARY_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}")
-
+set(SOF_ALSA_TOOLS_DIR "${SOF_ROOT_SOURCE_DIRECTORY}/../tools/bin")
+set(ALSATPLG_CMD "${SOF_ALSA_TOOLS_DIR}/alsatplg")
```

Comment:
It seems this is passing Windows CI runs, so apparently there's no attempt to run this (which is good).

We do need to add a call to new build script somewhere in our build flows, and somewhere that is specific to Linux. E.g. our SOF jenkins build now failing:
https://[sof-ci.01.org/sofpr/PR9870/build11207/build/topology/tplg.log](https://sof-ci.01.org/sofpr/PR9870/build11207/build/topology/tplg.log)

CMake Warning at topology/CMakeLists.txt:16 (message):
  /home/sof/work/sof.git/tools/../../tools/bin/alsatplg --version returned
  status: No such file or directory,

Reply:
This is due to the missing alsa-utils patch

Reply:
fixed that part for github workflows that doen't use Docker